docs/user_guide/4_FAQ.md

```diff
@@ -2,38 +2,97 @@
-* Check you’re using the right interface
+**Q: Can I fix the Hand myself?**
 
-## Software Issues
-### Docker
+A: We provide a set of tools that allow you to re-tension the tendons and complete some other tasks on the Hand yourself, so it won’t have to be sent back to Shadow. Where possible, one of our support staff will organise a video call with you to talk you through how to fix any problems you may have. For more complex diagnostic issues, the Hand will need to be sent back to Shadow.
+
+
+
+## Software
+
+**Q: The Hand doesn’t react after startup of the container**
+
+A: Check that:
+* The interface ID is set correctly
+* You have set the correct branch for sr_config
+If it is still not responding, power cycle the Hand and try again.
+
+**Q: Why is the control loop now running on the NUC? Can I run the system without the NUC if I prefer?**
+
+A: The NUC was introduced to avoid cycle skips for certain setups. This is more common in our more complex systems when we are running other heavy programs on the same laptop. If you are running just the hand locally you may not experience this issue, but you will start to see overruns if running other computationally expensive software. We strongly advise that the Hand is run using the NUC to ensure best performance.
+
+However, using the NUC is not mandatory and no changes have been made to the firmware. We have provided a set of icons that are in a Desktop folder called “Shadow Advanced Launchers”. One of these icons, called “Launch Local Shadow Hand”, will allow you to launch and run the hand connected locally to your laptop. More information on the Advanced Launch icons can be found [here](https://dexterous-hand.readthedocs.io/en/master/user_guide/1_2_setting_up_a_real_hand.html#understanding-the-icons-on-the-server-laptop).
+
+**Q: How does the NUC fit in with secondary development for the Hand?**
+
+A: The NUC only handles low level communication with the Hand as it is only running the driver. If you would like to do your own development with the Hand, continue doing so on the laptop and your scripts will be able to control the Hand. If you would like to change something on the NUC side, the icon called “Launch NUC Container” (within the Shadow Advanced Launchers desktop folder) will give you quick access to the NUC and provide a terminal with full access to the driver. 
+
+**Q: Why do you use Docker?**
+
+A: Using Docker helps us with customer support and development as we are able to replicate our customers’ exact software environment if problems arise. If you would like to start the container within Docker without starting the whole system, this can be done using the “1 - Launch Server Container” and “Launch NUC Container” icons within the Shadow Advanced Launchers desktop folder.  More information on the Advanced Launch icons can be found [here](https://dexterous-hand.readthedocs.io/en/master/user_guide/1_2_setting_up_a_real_hand.html#understanding-the-icons-on-the-server-laptop)..
```

Comment:
```suggestion
A: Using Docker helps us with customer support and development as we are able to replicate our customers’ exact software environment if problems arise. If you would like to start the container within Docker without starting the whole system, this can be done using the “1 - Launch Server Container” and “Launch NUC Container” icons within the Shadow Advanced Launchers desktop folder.  More information on the Advanced Launch icons can be found [here](https://dexterous-hand.readthedocs.io/en/master/user_guide/1_2_setting_up_a_real_hand.html#understanding-the-icons-on-the-server-laptop).
```